projects/approved-premises-and-delius/README.md

```diff
@@ -1,3 +1,18 @@
-# approved-premises-and-delius
+# Approved Premises and Delius
 
-// TODO Describe the service
\ No newline at end of file
+Service that responds to Approved Premises domain events, calls the
+[HMPPS Approved Premises API](https://github.com/ministryofjustice/hmpps-approved-premises-api)
+and records referral progress against the probation case record in Delius. We
+also provide an API to give probation case information to support making
+approved premises referrals.
+
+## Business Need
+
+Supporting the Approved Premises service with the correct information required
+to make an Approved Premises application and ensuring the Probation
+Practitioner has an up-to-date view of activity in the Approved Premises
+service when interacting with the probation case via Delius.
+
+## Context Maps
+
+![Context Maps](./img/approved-premises-and-delius-context-maps.png)
```

Comment:
You'll need to put the images somewhere in `tech-docs/source/` - otherwise the paths end up wrong and the image doesn't load.  Just an artifact of hacking the README into the tech-docs 😄 

See for example: https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/projects/workforce-allocations-to-delius/README.md?plain=1#L17

Other than that, the content looks good 👍